src/Nethermind/Nethermind.Evm/Tracing/ITxTracer.cs

```diff
@@ -162,13 +162,12 @@ public interface ITxTracer : IWorldStateTracer, IDisposable
     /// <summary>
     ///
     /// </summary>
-    /// <param name="depth"></param>
+    /// <param name="env"></param>
     /// <param name="gas"></param>
     /// <param name="opcode"></param>
     /// <param name="pc"></param>
-    /// <param name="isPostMerge"></param>
     /// <remarks>Depends on <see cref="IsTracingInstructions"/></remarks>
-    void StartOperation(int depth, long gas, Instruction opcode, int pc, bool isPostMerge = false);
+    void StartOperation(in ExecutionEnvironment env, long gas, Instruction opcode, int pc);
```

Comment:
I wonder is the order here most user-friendly or should it be something more like:
opcode, pc, gas, env
or 
pc, opcode, gas, env
?

Not sure to be honest which one would be best (including current one)

Reply:
Yeah good point, I think that `pc, opcode, gas, env` probably makes the most sense from a reader's perspective. I'll update this!